<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>public_key_Form</class>
 <widget class="QWidget" name="public_key_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="2" column="0">
    <widget class="public_pushbutton_numkey" name="pushButton_num7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="public_pushbutton_numkey" name="pushButton_num9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="public_pushbutton_numkey" name="pushButton_dot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="public_pushbutton_numkey" name="pushButton_num5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_clear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="public_pushbutton_numkey" name="pushButton_num1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="public_pushbutton_numkey" name="pushButton_num0">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="public_pushbutton_numkey" name="pushButton_num3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="public_pushbutton_numkey" name="pushButton_num4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="public_pushbutton_numkey" name="pushButton_num2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="public_pushbutton_numkey" name="pushButton_num6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="public_pushbutton_numkey" name="pushButton_num8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>pushbutton_key</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>public_pushbutton_numkey</class>
   <extends>QPushButton</extends>
   <header>public_pushbutton_numkey.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
